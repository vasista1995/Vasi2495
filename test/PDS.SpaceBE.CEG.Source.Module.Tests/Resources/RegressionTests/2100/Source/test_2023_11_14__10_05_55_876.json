{
  "Plant": "0600",
  "IdSystemName": "HPSModFAUF",
  "Line": "Econo",
  "ProductionOrder": "81770052",
  "Batch": "HK4919P0",
  "MaterialNumber": "43784",
  "ProductGroup": "M-ECONO",
  "MeasurementRecipe": "5",
  "ProcessEquipment": "10000570",
  "OrderType": "2100",
  "QrkGroup": "2100",
  "ProcessGroup": "system_soldering",
  "SpcClass": "2100",
  "ParameterType": "2100",
  "LongParameterName": "K2-Flutzeit HCOOH",
  "MaterialNumberText": "2100",
  "ProcessEquipmentName": "VADU7",
  "Operator": "2100",
  "ParameterClass": "NON_PPAP",
  "Segment": "HPS",
  "Package": "ECONO",
  "Module": "11 substrate-baseplate interconnect",
  "PosName": "2100",
  "FourDReport": "*",
  "VariableExtr3": "55",
  "VariableExtr4": "VADU7",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 332712354,
      "Seqnr": 1,
      "Value": 57.7,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "UniqueSourceKey": "CEG:BE:81770052:Econo:system soldering:K2-t-HCOOH:101065:332712354:0",
  "PKey": "CEG:BE:81770052:Econo:system soldering:K2-t-HCOOH:101065:332712354:0",
  "ParameterName": "K2-t-HCOOH",
  "ParameterUnit": "s",
  "SampleDate": "2023-10-27T11:16:54Z",
  "SampleDateUtc": "2023-10-27T09:16:54Z",
  "SampleSize": 1,
  "Sigma": "0",
  "Mean": "57,7",
  "Median": "57,7",
  "Min": "57,7",
  "Max": "57,7",
  "Q1": "57,7",
  "Q3": "57,7",
  "LdsID": "2100",
  "IsCalcParam": 0,
  "MeanCntrlLow": 50.2064100451012,
  "MeanCntrlTarget": 57.5775070290534,
  "MeanCntrlHigh": 64.9486040130056,
  "SpecLow": 42.0,
  "SpecTarget": 55.0,
  "SpecHigh": 68.0,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "N",
  "CtrlSigmaLowEnabled": "N",
  "CtrlSigmaCenterEnabled": "N",
  "CtrlRangeCenterEnabled": "N",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "N",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-10-27T11:22:09Z",
  "CreatedDateUtc": "2023-10-27T09:22:09Z",
  "UpdatedDate": "2023-10-27T11:22:09Z",
  "UpdatedDateUtc": "2023-10-27T09:22:09Z",
  "RvStoreFlag": "Y",
  "Operation": "system soldering",
  "ChannelId": "101065",
  "CkcId": "0",
  "ChannelName": "K2-t-HCOOH /Activation_target_55/VADU7",
  "SpaceInstanceName": "BE",
  "Sitekey": "CEG",
  "CalcCreated": "10/09/2023 17:20:42",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 58.3571428571429,
  "ExtEwmaMV": 57.9471260680603,
  "ExtMsMV": 0.353553390593274,
  "OutOfOrder": "N",
  "EwmaS": 0.0,
  "EwmaR": 0.0,
  "Calc2ID": 10002160,
  "LimitEnable": "YYNNNNYNNYYYYYYYYYYNNNYYYYYYNNNNNNNNN",
  "ExtEwmaMVLCL": 55.1199172464358,
  "ExtEwmaMVCenter": 57.5775070290534,
  "ExtEwmaMVUCL": 60.035096811671,
  "ExtMaMVLCL": 55.7197437749336,
  "ExtMaMVCenter": 57.5775070290534,
  "ExtMaMVUCL": 59.4352702831732,
  "EwmaSLCL": 0.0,
  "EwmaSCenter": 0.0,
  "EwmaSUCL": 0.0,
  "EwmaRLCL": 0.0,
  "EwmaRCenter": 0.0,
  "EwmaRUCL": 0.0,
  "ExtMSMVLCL": 0.00173930510205841,
  "ExtMSMVCenter": 0.820204066545146,
  "ExtMSMVUCL": 3.29479149911247,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 50.2064100451012,
  "RawUal": 64.9486040130056,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND 4.5 Sigma",
  "ChannelType": "N",
  "ChState": "Online"
}