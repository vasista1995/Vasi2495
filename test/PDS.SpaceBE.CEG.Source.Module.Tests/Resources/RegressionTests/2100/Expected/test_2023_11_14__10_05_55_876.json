{
  "IdSource": "SPACEACT3:CEG:BE:81770052:Econo:system soldering:K2-t-HCOOH:101065:332712354:0",
  "Data_flat_Metadata": {
    "Plant": "0600",
    "Line": "Econo",
    "ProductionOrder": "81770052",
    "Batch": "HK4919P0",
    "MaterialNumber": "43784",
    "ProductGroup": "M-ECONO",
    "MeasurementRecipe": "5",
    "ProcessEquipment": "10000570",
    "OrderType": "2100",
    "QrkGroup": "2100",
    "ProcessGroup": "system_soldering",
    "SpcClass": "2100",
    "ParameterType": "2100",
    "LongParameterName": "K2-Flutzeit HCOOH",
    "MaterialNumberText": "2100",
    "ProcessEquipmentName": "VADU7",
    "Operator": "2100",
    "ParameterClass": "NON_PPAP",
    "Segment": "HPS",
    "Package": "ECONO",
    "Module": "11 substrate-baseplate interconnect",
    "PosName": "2100",
    "FourDReport": "*",
    "ParameterName": "K2-t-HCOOH",
    "ParameterUnit": "s",
    "ParameterOper": "system soldering",
    "SiteKey": "CEG",
    "ChannelId": "101065",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-27T09:22:09Z",
    "CreatedTimestamp": "2023-10-27T11:22:09Z",
    "UpdatedTimestamp": "2023-10-27T11:22:09Z",
    "UpdatedTimestampUtc": "2023-10-27T09:22:09Z",
    "ExportedTimestampUtc": "2023-10-27T11:16:54Z",
    "ExportedTimestamp": "2023-10-27T11:16:54Z",
    "SampleTimestampUtc": "2023-10-27T09:16:54Z",
    "SampleTimestamp": "2023-10-27T11:16:54Z",
    "SourceDataLevel": "C",
    "ChannelName": "K2-t-HCOOH /Activation_target_55/VADU7",
    "RvStoreFlag": "Y",
    "CalcCreated": "10/09/2023 17:20:42",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 10002160,
    "ViolCount": 0,
    "CalculationStrategy": "ND 4.5 Sigma",
    "LdsID": "2100",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 68.0,
        "SpecLow": 42.0,
        "SpecTarget": 55.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNNNNYNNYYYYYYYYYYNNNYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "57,7",
        "Median": "57,7",
        "Sigma": "0",
        "Min": "57,7",
        "Max": "57,7",
        "Q1": "57,7",
        "Q3": "57,7"
      },
      "ControlLimits": {
        "ctrl_mean_low": 50.2064100451012,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 57.5775070290534,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 64.9486040130056,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 58.3571428571429,
        "ExtEwmaMV": 57.9471260680603,
        "ExtMsMV": 0.353553390593274,
        "EwmaS": 0.0,
        "EwmaR": 0.0,
        "ExtEwmaMVLCL": 55.1199172464358,
        "ExtEwmaMVCenter": 57.5775070290534,
        "ExtEwmaMVUCL": 60.035096811671,
        "ExtMaMVLCL": 55.7197437749336,
        "ExtMaMVCenter": 57.5775070290534,
        "ExtMaMVUCL": 59.4352702831732,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.00173930510205841,
        "ExtMSMVCenter": 0.820204066545146,
        "ExtMSMVUCL": 3.29479149911247,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 50.2064100451012,
        "RawUal": 64.9486040130056,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 332712354,
        "Seqnr": 1,
        "Value": 57.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 1
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CEG:BE:Econo::K2-t-HCOOH:101065",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "HPSModFAUF:81770052",
    "IdSystemName": "HPSModFAUF",
    "Type": "Fauf"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CEG:BE:81770052:Econo:system soldering:K2-t-HCOOH:101065:332712354:0"
  }
}